storybook/src/stories/UseMutationObserver/Attributes.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+import { useMutationObserver } from 'too-many-hooks'
+
+interface Props {
+  attributeMutation: (name?: string | null) => string
+  attribute: string
+}
+
+const Attributes: React.FC<Props> = ({ attributeMutation, attribute }) => {
+  const record = useMutationObserver(document.body, {
+    attributes: true,
+    subtree: true,
+  })
+  useEffect(() => {
+    const exampleDiv = document.getElementById('example-div')
+    attributeMutation(
+      `Observed mutation with record attributeName ${record?.attributeName}, div data set to ${exampleDiv?.dataset.attribute}`,
+    )
+  }, [attributeMutation, record])
```

Comment:
This effect runs on the initial render when the mutation observer is initially defined. We should have some check that these values are defined before calling the action in order to prevent this:

<img width="805" alt="image" src="https://user-images.githubusercontent.com/10625031/215298798-4ec59b27-f02b-45ca-9864-ca685d632ee3.png">